src/compiler/transformers/declarations.ts

```diff
@@ -251,7 +251,11 @@ const declarationEmitNodeBuilderFlags = NodeBuilderFlags.MultilineObjectLiterals
  *
  * @internal
  */
-export function transformDeclarations(context: TransformationContext) {
+export function transformDeclarations(context: TransformationContext): {
+    (node: Bundle): Bundle;

```

Comment:
This seems weird; do we not have a type for this somewhere? I think this is a regular transformer? I'd expect this to have the same signature as other transformers...